lib/ret/account.ex

```diff
@@ -38,6 +38,10 @@ defmodule Ret.Account do
   def has_admin_accounts?(), do: from(a in Account, limit: 1) |> where(is_admin: true) |> Repo.exists?()
   def exists_for_email?(email), do: account_for_email(email) != nil
 
+  def get_account_by_id(account_id) do
+    Repo.get(Account, account_id)
```

Comment:
This isn’t the first use of `Repo` inside this module, but this is a good opportunity to do the right thing. A good pattern for managing state is: functional core, imperative shell. Struct manipulation is low-level, which make stateful calls (e.g., `Repo` calls) within a struct module a design smell.

How to fix? Reserve this module for pure functions that operate on `Ret.Account.t()` and put any new repo calls in the context boundary module.